### PR TITLE
docs: add missing default slots

### PR DESCRIPTION
**Pull request by benelan**
**Related Issue:** #2974

## Summary
Added documentation for missing default slots

### CODE REVIEW

File: src/components/calcite-label/calcite-label.tsx [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { FocusRequest } from "./interfaces";
 import { Alignment, Scale, Status } from "../interfaces";
 import { CSS_UTILITY } from "../../utils/resources";
 
+/**
+ * @slot - A slot for adding text.
```
Comment (driskull) [inline src/components/calcite-label/calcite-label.tsx:8]: I think we should say text and a labelable component. We might need to document the labelable components here as well. thoughts @jcfranco ?

**Review by driskull — APPROVED** [review]
👍

File: src/components/calcite-label/calcite-label.tsx (hunk 1 above)
Comment (benelan) [reply src/components/calcite-label/calcite-label.tsx:8]: @driskull do you mean a native label?

Comment (driskull) [reply src/components/calcite-label/calcite-label.tsx:8]: No, one of our components that supports a calcite-label.

**Pull request merged**